src/Common/src/Microsoft/Win32/SafeHandles/SafeEvpCipherCtxHandle.Unix.cs

```diff
@@ -37,11 +37,11 @@ internal static SafeEvpCipherCtxHandle Create()
 
             try
             {
-                memPtr = Marshal.AllocHGlobal(Interop.libcrypto.EVP_CIPHER_CTX_SIZE);
+                memPtr = Marshal.AllocHGlobal(Interop.Crypto.EVP_CIPHER_CTX_SIZE);
```

Comment:
Actually, while you're in here; could you take care of #1963, and make Create and Free methods for this type?  Then we can get rid of EVP_CIPHER_CTX_SIZE.